include/bluetooth/peer_manager/peer_manager.h

```diff
@@ -241,15 +241,16 @@ bool pm_sec_is_sufficient(uint16_t conn_handle, pm_conn_sec_status_t *p_sec_stat
  *
  * @note The key must continue to reside in application memory as it is not copied by Peer Manager.
  *
- * @note This function is deprecated. LESC keys are now handled internally if @ref PM_LESC_ENABLED
- *       is true. If @ref PM_LESC_ENABLED is false, this function works as before.
+ * @note This function is deprecated. LESC keys are now handled internally if the @c CONFIG_PM_LESC
+ *       Kconfig option is enabled. If the @c CONFIG_PM_LESC Kconfig option is disabled, this
+ *       function works as before.
```

Comment:
What do you mean by this last sentence?
```
 If the @c CONFIG_PM_LESC Kconfig option is disabled, this function works as before.
```
If this is still meant to be used when CONFIG_PM_LESC Kconfig option is disabled, this is not deprecated.